scripts/gen_profiles_layer.py

```diff
@@ -561,8 +549,9 @@ class JsonLoader {
     }
 
     static void Store(VkInstance instance) {
-        profile_map()[instance] = profile_map()[VK_NULL_HANDLE];
-        profile_map().erase(VK_NULL_HANDLE);
+        auto nh = profile_map().extract(VK_NULL_HANDLE);
+        nh.key() = instance;
+        profile_map().insert(std::move(nh));
```

Comment:
Sorry but unless there is actual proof it makes a significant benefit, I am strongly opposed in using cool C++ features instead of good old C++ understandable by anyone.

Reply:
This is a major performance benefit. What the 'good old understandable' code does is copy the entire structure to a different key, which is a large struct containing many vectors & maps. What the new code does is "change the key the struct is associated with, no copies".

The CMake code specifies C++17 and so makes this a viable option, since the `extract()` function was added in C++17.